package.json

```diff
@@ -90,7 +90,7 @@
     "report-latency": "./bin/report-latency"
   },
   "engines": {
-    "node": ">=8.3.0"
+    "node": ">=10.21.0"
```

Comment:
Please add a migration guide like [6to7](https://github.com/restify/node-restify/blob/master/docs/guides/6to7guide.md) and highlight Node version bump.

Reply:
Should the migration guide be a separate PR?

Reply:
yes, pleace